apps/daimo-mobile/src/view/screen/HistoryOpScreen.tsx

```diff
@@ -75,32 +64,46 @@ function LinkToExplorer({ txHash }: { txHash: string }) {
 }
 
 function TransferBody({ op }: { op: TransferOpEvent }) {
+  const [account] = useAccount();
+  assert(account != null);
+
+  const [directionSymbol, directionText] = (() => {
+    if (op.from === account.address) {
+      return [
+        "↗",
+        <>
+          to <AddrText addr={op.to} />
+        </>,
+      ];
+    } else {
+      return [
+        "↘",
+        <>
+          from <AddrText addr={op.from} />
+        </>,
+      ];
+    }
+  })();
+
   return (
     <View>
-      <Spacer h={32} />
       <TitleAmount amount={BigInt(op.amount)} />
-      <Spacer h={24} />
-      <View style={styles.kvList}>
-        <KVRow k="From" v={<AddrText addr={op.from} />} />
-        <KVRow k="To" v={<AddrText addr={op.to} />} />
-        <KVRow k="Date" v={timeString(op.timestamp)} />
-        {op.opHash && <KVRow k="User op" v={op.opHash} />}
-        {op.blockNumber && <KVRow k="Block" v={op.blockNumber} />}
-      </View>
-    </View>
-  );
-}
-
-function KVRow({ k, v }: { k: string; v: React.ReactNode }) {
-  return (
-    <View style={styles.kvRow}>
-      <View style={styles.kvKey}>
-        <TextLight numberOfLines={1}>{k}</TextLight>
-      </View>
-      <View style={styles.kvVal}>
-        <TextBody numberOfLines={1}>
-          <TextBold>{v}</TextBold>
-        </TextBody>
+      <Spacer h={16} />
```

Comment:
may be worth putting space around the title

Reply:
doubled to 32